packages/generator-react-server/generators/app/templates/test.js

```diff
@@ -0,0 +1,54 @@
+import cp from 'child_process';
+import http from 'http';
+import test from 'ava';
+
+test.before('start the server', async () => {
+	let stdout = await exec('npm run compile');
+	console.log(stdout);
+	stdout = await exec('node node_modules/.bin/pm2 start npm -i 1 --name "hello-world" -- start');
```

Comment:
I'm not sure how I would bring the server down when the tests are done, or how to make it possible to tail the logs in real time in a separate window.  We may very well be able to overcome both of those when `exec`ing the server directly; `pm2` is not a small dep, and I'd be happy to get rid of it.